internal/ethapi/api.go

```diff
@@ -1555,7 +1549,12 @@ func (s *PublicTransactionPoolAPI) SendRawTransaction(ctx context.Context, encod
 	if err := rlp.DecodeBytes(encodedTx, tx); err != nil {
 		return common.Hash{}, err
 	}
-	meta := types.NewTransactionMeta(nil, nil, types.SighashEIP155, types.QueueOriginSequencer)
+
```

Comment:
Need to add logic here for using a timestamp of:

 `max(force inclusion perion - 5 mins, s.b.GetLatestTimestamp() )`

Reply:
Could query for the force inclusion period at runtime if it is made public here: https://github.com/ethereum-optimism/contracts-v2/blob/c9d77a2e6bcbd71c1de56305750b80346bd8ba37/contracts/optimistic-ethereum/OVM/chain/OVM_CanonicalTransactionChain.sol#L55

Reply:
Ah should this comment be updated with the new -- `uint64(time.Now().Add(-5 * time.Minute).Unix())` ?